samples/sample_multi_transcode/include/pipeline_transcode.h

```diff
@@ -571,8 +571,10 @@ namespace TranscodingSample
         std::mutex                   m_mutex;
         std::list<SurfaceDescriptor> m_SList;
         bool                         m_IsBufferingAllowed;
-        MSDKEvent*                   pRelEvent;
-        MSDKEvent*                   pInsEvent;
+        std::condition_variable      m_RelEvent;
```

Comment:
How often we use these CVs? Windows has a bug in CV implementation which causes significant increase in CPU%... We might need to be careful with this PR...

Reply:
I'm afraid quite often. For example they are used in 1:N scenarios for synchronization of produce/consume parts of pipelines. Which windows implementation bug do you refer?